docs/storage.md

```diff
@@ -179,6 +189,38 @@ Now you can create the SR where `server` is your mon ip.
 * You may specify `serverport` option if you are using any other port than 6789.
 :::
 
+### Ceph iSCSI gateway
```

Comment:
I know you'll enrich the section after testing. It should probably mention that the SR type would be `lvmoiscsi`

Reply:
Mentioned `lvmoiscsi` driver.